src/lib/stores/Resources.ts

```diff
@@ -0,0 +1,59 @@
+import { get, writable, type Writable } from 'svelte/store';
+
+import * as api from '$lib/api';
+import { Picsure } from '$lib/paths';
+import { features } from '$lib/configuration';
+import { toaster } from '$lib/toaster';
+
+interface QueryResource {
+  name: string;
+  uuid: string;
+}
+
+export interface ResourceMap {
+  visualization: string;
+  application: string;
+  aggregate: string;
+  hpdsOpen: string;
+  hpdsAuth: string;
+  query: QueryResource[];
```

Comment:
I think query is too overloaded of a term

Reply:
That's fair. Suggestions for a better term?